src/AlgebraicGeometry/Schemes/Sheaves/IdealSheaves.jl

```diff
@@ -345,6 +345,7 @@ end
 end
   
 @attr Bool function has_dimension_leq_zero(I::Ideal)
+  is_one(I) && return true
```

Comment:
This was just removed in https://github.com/oscar-system/Oscar.jl/pull/4177/files#diff-0fb0ac851cccf4c4419c89353ba64ef43364db228c65357487fe512e76917062L348. Maybe check if you re-added more things by accident

Reply:
I know... It happened in the rebase. But then I thought: "Well, maybe @simonbrandhorst was right and there was a shortcut in some cases where the `:is_one` attribute was set?" So it also doesn't hurt to keep it.

Reply:
Keep it. In the worst case, it computes and stores the GB properly implying that the dimension further below is just a lightweight combinatorial computation on the leading terms.